checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
```

Comment:
@brianyu28 can we simplify this to `from check50 import *`, so that authors don't need to remember what to import? Or will that pull in too much?